tests/test_patch.py

```diff
@@ -49,3 +52,40 @@ def test_distributed():
     import distributed
 
     assert hasattr(distributed, "_rapids_patched")
+
+
+@pytest.mark.parametrize("python_version", [(3, 11, 9), (3, 11, 8)])
+@run_test_in_subprocess
+def test_dask_accessor(python_version):
+    import sys
+
+    import dask
+
+    # Simulate the version of Python and Dask needed to trigger vendoring of the
+    # accessor module.
+    sys.version_info = python_version
+    dask.__version__ = "2023.4.1"
+
+    from dask.dataframe import accessor
+
+    assert (hasattr(accessor, "_rapids_vendored")) == (python_version >= (3, 11, 9))
+
+
+def test_dask_cli():
+    try:
+        subprocess.run(["dask", "--help"], capture_output=True, check=True)
+    except subprocess.CalledProcessError as e:
+        print(e.stdout.decode())
+        print(e.stderr.decode())
+        raise
+
+
+def test_dask_as_module():
```

Comment:
Nice. Maybe we can also test the `distributed.cli.dask_spec` case in the original issue?

Reply:
Sure.